app/controllers/members_controller.rb

```diff
@@ -1,8 +1,17 @@
 class MembersController < ApplicationController
-
   def create
     event = Event.find(params[:event_id])
     event.members << current_user
     redirect_back(fallback_location: root_path)
-  end 
-end
+  end
+
+  def destroy
+    event = Event.find(params[:event_id])
+    event.members = current_user
```

Comment:
You don't what to assign `current_user` to members. You want to remove it from members. And to do that you need to check if you have permission to do that. Like we discusses what should happen when user is member or owner of event.